batchflow/models/metrics/classify.py

```diff
@@ -158,6 +159,35 @@ def __getattr__(self, name):
     def confusion_matrix(self):
         return self._confusion_matrix.sum(axis=0)
 
+    def show_confusion_matrix(self, classes=None, normalize=False):
+        """ Show confusion matrix.
+
+        Parameters
+        ----------
+        classes : sequence, optional
+            Sequence of classes labels.
+        normalize : bool
+            Whether to normalize confusion matrix over target classes.
+        """
+        confusion_matrix = np.array(self.confusion_matrix)
+        if classes is None:
+            classes = np.arange(self.num_classes)
+        if normalize:
+            confusion_matrix = confusion_matrix/np.nansum(confusion_matrix, axis=0)
+
+        fig, ax = plt.subplots(figsize=(10, 10))
```

Comment:
why do you hardcode figsize?

Reply:
I added it to parameters from now, but I am not sure that it is the only parameter we need to add.